src/widgets/breadcrumb/__tests__/breadcrumb-test.js

```diff
@@ -109,7 +109,9 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/breadcrumb/
         instantSearchInstance: {},
       });
 
-      expect(render.mock.calls[0][0]).toMatchSnapshot();
+      const [firstRender] = render.mock.calls;
+
+      expect(firstRender[0].props).toMatchSnapshot();
```

Comment:
shouldn't this be done by test-renderer like the other tests?

Reply:
WDYM?

Reply:
I misunderstood what these tests are doing, i thought they checked the output, but instead it's just asserting the props. I think we can remove the tests fully (in a stacked PR), wdyt?

Reply:
At some point we want to remove snapshots at this level. This is however not the only widget using those (all of them use snapshots).